src/models/db/notification.js

```diff
@@ -1,18 +0,0 @@
-'use strict'
-
-const Model = require('five-bells-shared').Model
-const PersistentModelMixin = require('five-bells-shared').PersistentKnexModelMixin
-const knex = require('../../lib/knex').knex
-
-class Notification extends Model {
```

Comment:
The `notifications` and `subscriptions` tables are basically gone. Once we update a transfer, the ledger will send a notification through websocket and then forget about it. By design, these notifications aren't robust. It's up to the client to send a request for a notification to be resent if it didn't get it.

Robust notifications in the ledger are very expensive in terms of transactions (over more than one table). We may want to reimplement a robust notifications using AMQP or something like that. That would move some complexity to another specialized subsystem (a queue).